recipes/pyutilsnrw/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyutilsnrw" %}
+{% set version = "0.1.0" %}
+{% set hash_type = "md5" %}
+{% set hash_value = "286394472d5a7a24ff647db5b9020a98" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+  skip: True  # [py34]
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
```

Comment:
It looks like only `python` and `setuptools` is required to build the package since we're using `--single-version-externally-managed`, so please remove the others.